packages/bpk-component-close-button/src/BpkCloseButton.module.scss

```diff
@@ -50,4 +50,16 @@ $button-padding: tokens.bpk-spacing-sm() * 0.5;
     vertical-align: -1.75 * $button-padding;
     fill: currentcolor;
   }
+
+  &--on-dark {
+    color: tokens.$bpk-text-primary-inverse-day;
+
+    @include utils.bpk-hover {
+      color: tokens.$bpk-text-primary-inverse-day;
+    }
+
+    &:active {
+      color: tokens.$bpk-text-primary-inverse-day;
+    }
+  }
```

Comment:
If we separate the classes into common and variant it can make the consuming code a little nicer: 

```scss
.common {
// all common styles
}

.default {
   composes: common;
   color: tokens.$bpk-text-primary-day;
   //hover & active too
}

.onDark {
   composes: common;
   color: tokens.$bpk-text-primary-inverse-day;
   //hover & active too
}
```

The in the component:
```jsx
const className = props.onDark ? STYLES["onDark"] : STYLES["default"];

```

Reply:
Composes is css modules btw: https://github.com/css-modules/css-modules/blob/master/docs/composition.md

Reply:
Ohh nice! Didn't know about this.

Reply:
You might need a small modification to the linting setup (as I think by default it doesn't understand `composes`)

Reply:
Looks like out current style-lint doesn't allow us to use composes :
```
✖ stylelint --fix:

packages/bpk-component-close-button/src/BpkCloseButton.module.scss
 46:5  ✖  Unexpected unknown property "composes"  property-no-unknown
```

Reply:
Updated, stylelint rules, had to keep the naming convention though so it still looks like before but makes uses of composes.